packages/gasket-nextjs/src/index.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Main, NextScript } from 'next/document';
 import htmlescape from 'htmlescape';
+import { useGasketData as _useGasketData } from './use-gasket-data';
+import { withGasketDataProvider as _withGasketDataProvider } from './with-gasket-data-provider';
```

Comment:
If these are written more consicely using export from, does that solve your use case?
```suggestion
export { useGasketData } from './use-gasket-data';
export { withGasketDataProvider } from './with-gasket-data-provider';
```

Reply:
unfortunately it dosent. With this approach babel does the same thing as the original way i had it. it converts the exports to getter functions and those dont place nice with stubbing